sensor/upgrader/preflight/access.go

```diff
@@ -94,10 +98,16 @@ func (c accessCheck) Check(ctx *upgradectx.UpgradeContext, execPlan *plan.Execut
 			reporter.Warnf("Evaluation error performing access review check for action %s on resource %s: %s", ra.Verb, ra.Resource, sarResult.Status.EvaluationError)
 		}
 		if !sarResult.Status.Allowed && !sarResult.Status.Denied {
-			reporter.Errorf("K8s authorizer did not explicitly allow or deny access to perform action %s on resource %s. This usually means access is denied.", ra.Verb, ra.Resource)
+			actionResourceErr = append(actionResourceErr, fmt.Sprintf("%s: %s", ra.Verb, ra.Resource))
 		} else if !sarResult.Status.Allowed {
 			reporter.Errorf("Action %s on resource %s is not allowed", ra.Verb, ra.Resource)
 		}
 	}
+	if len(actionResourceErr) > 0 {
+		reporter.Errorf("K8s authorizer did not explicitly allow or deny access to perform "+
+			"the following actions on following resources: %s. This usually means access is denied.",
+			strings.Join(actionResourceErr, ","))
```

Comment:
Good idea. Just a nit regarding the actual format produced by this...

In l101 you produce strings of the form `"VERB: RES"`. In l109 these are joined to produce something like this `"VERB1: RES1,VERB2: RES2,VERB3: RES3"`, right?

I'm just imagining that the whitespace we have between the verb and the resource might make it harder to identify the actual pairs belonging together. I mean, on first sight I would probably associate "RES1" with "VERB2", etc. instead of "VERB2" with "RES2".

But I haven't seen this in practice. So, if you think this is sufficiently good to read, then. :+1:

Thank you.

Reply:
This is a good point. Let me see if I can improve this aspect

Reply:
[0db6e62](https://github.com/stackrox/stackrox/pull/9928/commits/0db6e62785568546c2543c2a1e4b20123ad10a6d) should address those issues. Let me know what you think

Reply:
sorry, totally missed this. Thanks, looking good.